pkgs/top-level/aliases.nix

```diff
@@ -414,6 +414,8 @@ mapAliases ({
   godot-headless = throw "godot-headless has been renamed to godot3-headless to distinguish from version 4"; # Added 2023-07-16
   godot-server = throw "godot-server has been renamed to godot3-server to distinguish from version 4"; # Added 2023-07-16
 
+  gdtoolkit = throw "gdtoolkit has been renamed to gdtoolkit_3 to distinguish from version 4"; # Added 2024-02-17
```

Comment:
whats the justification for now having this toplevel attribute ?

Reply:
Like with the package name, I followed how this was implemented for Godot 3 and 4.

https://github.com/NixOS/nixpkgs/blob/614b4613980a522ba49f0d194531beddbb7220d3/pkgs/top-level/aliases.nix#L327